pyodide-build/.gitignore

```diff
@@ -0,0 +1,11 @@
+*.pyc
```

Comment:
We already have a top level .gitignore which should take care of these?

Reply:
Right, good point.